pkgs/development/compilers/zig/0.10.nix

```diff
@@ -8,15 +8,15 @@
 , zlib
 }:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: {
```

Comment:
I added finalattrs to zig 0.10 and 0.9

Reply:
I think what @AndersonTorres is referring to is the ability to use this pattern with `buildZigPackage`, so you should be able to do something like this

```nix
zig.buildZigPackage (finalAttrs: {
  ...
})
```

Reply:
Precisely what figsoda said.

Reply:
I can't figure out how to do this yet

Reply:
Roberth gave advice before about this.